stories/components/FsForm.vue

```diff
@@ -54,14 +54,14 @@ FForm.FForm(
         name="FTextarea"
         hint="Hint for FTextarea"
       )
-    FGridItem.FForm__actions(span="12")
+    FGridItem.FsForm__actions(span="12")
       FButton(@click="submit") Submit
-      FButton(@click="resetForm") Reset
+      FButton(@click="() => reset()") Reset
```

Comment:
ça ne fonctionne pas, simplement ça ?
```suggestion
      FButton(@click="reset") Reset
```

Reply:
Techniquement si, ça marche ! Mais avec une erreur TS : 
> Type '(state?: Partial<FormState<Record<string, unknown>>> | undefined) => void' is not assignable to type '(e: MouseEvent | KeyboardEvent) => any'.
>   Types of parameters 'state' and 'e' are incompatible.ts(2322)

Le reset prend un state en params mais le `FButton` emit l'event en passant l'event :/ 
C'est un petit contournement du TS mais je me dis finalement que ça évite tout problème de passer un event qui potentiellement aurait des champs en commun avec le state.

Qu'en penses-tu ?

Reply:
Oui ok très bien ;-)